docs/wallets/helium-wallet-app.mdx

```diff
@@ -131,10 +136,12 @@ The Helium Wallet will display the token balances and fiat value in your account
 **HNT** - in USD or your defined currency  
 **MOBILE** - in USD or your defined currency  
 **IOT**- in USD or your defined currency  
-**DC**- in USD (Not transferable)  
-**SOL**- in USD or your defined currency
+**DC** - in USD (Not transferable)  
+**SOL** - in USD or your defined currency **USDC** - in USD or your defined currency
```

Comment:
Are these meant to be on two lines?

Reply:
No, it is supposed to go under, but I can't seem to make a hard enter line.

Reply:
Ok, I figured sth out.